### PR TITLE
[Examiner] data framework

### PR DESCRIPTION
**Pull request by kongtiaowang**
## Brief summary of changes
rewrite the Examiner module with data framework

### CODE REVIEW

File: modules/examiner/php/addexaminer.class.inc [hunk 1]
```diff
@@ -0,0 +1,173 @@
...
+                'fullName' => $fullName,
+                'siteID'   => $siteID,
+            ]
+        );
+
+        if (empty($examinerID)) {
+            return false;
+        } else {
+            return true;
+        }
```
Comment (CamilleBeau) [inline modules/examiner/php/addexaminer.class.inc:154]: I know this was already in the old code, but can be shortened with 
```suggestion
        return !empty($examinerID);
```

**Review by CamilleBeau — COMMENTED** [review]
Some small comments, but the code looks good!

**Comment by charliehenrib** [comment]
@CamilleBeau The site filtering is not rendering for me when I test it on your VM.

**Review by CamilleBeau — APPROVED** [review]
Looks good to me! @charliehenrib will be manually testing.

**Comment by charliehenrib** [comment]
Looks good for manual testing as well!

**Pull request merged**